### DISCUSSION TITLE
Webpacker can't find two_factor_authentication.js

### DISCUSSION THREAD

**Discussion by NotNite** [mastodon/mastodon#17640] [Q&A] [answered]
I can't log into my instance from another machine because I got logged out, and the two-factor authentication login page is the "This page is not correct" screen. In the logs, I see:

```
[593016ce-f91c-4802-8667-2dc9d0ab5b8d] ActionView::Template::Error (Webpacker can't find two_factor_authentication.js in /home/mastodon/live/public/packs/manifest.json. Possible causes:
1. You want to set webpacker.yml value of compile to true for your environment
   unless you are using the `webpack -w` or the webpack-dev-server.
2. webpack has not yet re-run to reflect updates.
3. You have misconfigured Webpacker's config/webpacker.yml file.
4. Your webpack configuration is not creating a manifest.
```

I've tried running precompile and restarting(/reloading) all services, but no dice. I'm running v3.4.6 without Docker, but the only thing I can think of that's weird is that I had to jump across some updates when v3.4.6 released. There's also a copy of my manifest in the logs, let me know if I should attach that.

**Comment by Gargron** [comment, marked as answer]
Try doing assets:clobber before assets:precompile. Maybe it’s a weird cached assets thing.

**Reply by NotNite** [reply]
Worked, thanks!
